src/test/java/bisq/core/dao/node/validation/GenesisTxValidatorTest.java

```diff
@@ -61,7 +61,7 @@ public void testGetGenesisTx() {
         // With mismatch in blockheight and tx id, we should not get genesis tx back.
         Optional<Tx> result = GenesisTxValidator.getGenesisTx(genesisTxId, genesisBlockHeight, genesisTotalSupply, rawTx);
         Optional<Tx> want = Optional.empty();
-        assertEquals(want, result);
+        Assert.assertEquals(want, result);
```

Comment:
We don't have a defined code style guide for static imports but usually if repeated calls we use static imports, specially if they are well known like Assert.